### PR TITLE
Changed Text to add more sizes

### PR DESCRIPTION
**Pull request by ericsoderberghp**
#### What does this PR do?

Changed Text to add more sizes

#### Where should the reviewer start?

Text -> All story

#### What testing has been done on this PR?

enhanced All story, enhanced unit tests

#### How should this be manually tested?

storybook

#### Do the grommet docs need to be updated?

automatic

#### Should this PR be mentioned in the release notes?

yes

#### Is this change backwards compatible or is it a breaking change?

backwards compatible

### CODE REVIEW

File: src/js/components/Text/doc.js [hunk 1]
```diff
@@ -74,6 +74,11 @@ export const doc = Text => {
         'large',
         'xlarge',
         'xxlarge',
+        '2xl',
```
Comment (ShimiSun) [inline src/js/components/Text/doc.js:77]: Line 76 seems like a duplicate of 77.

Comment (ericsoderberghp) [reply src/js/components/Text/doc.js:77]: 76 is backwards compatible, 77 is friendlier for the new naming, both seem reasonable to me

Comment (ShimiSun) [reply src/js/components/Text/doc.js:77]: seeing them both as consecutive sizes gives the impression that '2xl', is larger than 'xxlarge'.

Comment (ericsoderberghp) [reply src/js/components/Text/doc.js:77]: I believe it's somewhat standard in clothing sizes that "2XL" and "XXLARGE" mean the same thing. We can add a note in the documentation that they are identical.

**Review by ShimiSun — APPROVED** [review]

**Pull request merged**